extensions/sql/contract-definition-store-sql/src/test/java/org/eclipse/dataspaceconnector/sql/contractdefinition/store/SqlContractDefinitionStoreTest.java

```diff
@@ -11,6 +11,7 @@
  *       Daimler TSS GmbH - Initial Tests
  *       Microsoft Corporation - Method signature change
  *       Microsoft Corporation - refactoring
+ *       Microsoft Corporation - added tests
```

Comment:
Is there a reason for not having these three contributions in a single line?

Reply:
not a very strong one. my thinking was to have some sort of a chronology, once there are several contributors.